Gemfile

```diff
@@ -11,6 +11,7 @@ gem "bootsnap", "~> 1.18"
 gem "clearance", "~> 2.9"
 gem "dalli", "~> 3.2"
 gem "datadog", "~> 2.4"
+gem "debug"
```

Comment:
Intend for this to stay? move to dev probably?

Reply:
This is for dev (I'll probably introduce this as well soon)

Reply:
Removed